src/configuration.js

```diff
@@ -80,58 +54,58 @@ export default class Configuration {
         }
 
         if (type === 'remote') {
```

Comment:
I don't follow.  type is checked for in the previous function and why != 'remote'?